docs/operations/rule-configuration.md

```diff
@@ -23,212 +23,242 @@ title: "Retaining or automatically dropping data"
   -->
 
 
-In Apache Druid, Coordinator processes use rules to determine what data should be loaded to or dropped from the cluster. Rules are used for data retention and query execution, and are set via the [web console](./web-console.md).
+In Apache Druid, [Coordinator processes](../design/coordinator.md) use rules to determine what data to retain or drop from the cluster. 
 
-There are three types of rules, i.e., load rules, drop rules, and broadcast rules. Load rules indicate how segments should be assigned to different historical process tiers and how many replicas of a segment should exist in each tier.
-Drop rules indicate when segments should be dropped entirely from the cluster. Finally, broadcast rules indicate how segments of different datasources should be co-located in Historical processes.
+There are three types of rules: [load](#load-rules), [drop](#drop-rules), and [broadcast](#broadcast-rules). See the sections below for more information on each type.
 
-The Coordinator loads a set of rules from the metadata storage. Rules may be specific to a certain datasource and/or a
-default set of rules can be configured. Rules are read in order and hence the ordering of rules is important. The
-Coordinator will cycle through all used segments and match each segment with the first rule that applies. Each segment
-may only match a single rule.
+The Coordinator loads a set of rules from the metadata storage. You can configure a default set of rules to apply to all data sources, and/or you can set specific rules for specific data sources. 
 
-Note: It is recommended that the web console is used to configure rules. However, the Coordinator process does have HTTP endpoints to programmatically configure rules.
+## Set a rule
+
+To set a default retention rule for all data sources, send a POST request to the following API:
+
+`/druid/coordinator/v1/rules/_default`
+
+To set a retention rule for a specific data source, send a POST request to the following API:
+
+`/druid/coordinator/v1/rules/{dataSourceName}`
+
+The rules API accepts a list of rules. The payload you send in the API request for each rule is specific to the rules types outlined below.
+
+You can also set rules using the [web console](./web-console.md). Go into a data source and select **Actions** > **Edit retention rules**.
+
+### Rule order
+
+The order of rules is important. The Coordinator reads rules in the order in which they appear in the rules list. For example, in the following screenshot the Coordinator evaluates data against rule 1, then rule 2, then rule 3:
+
+![retention rules](../assets/retention-rules.png)
+
+In the web console you can use the up and down arrows on the right side of the interface to change the order of the rules.
+
+The Coordinator cycles through all used segments and matches each segment with the first rule that applies. Each segment can only match a single rule.
```

Comment:
in keeping with using API examples, you may want to include how to list rules:

```
curl --location --request GET 'http://localhost:8888/druid/coordinator/v1/rules'

```

Reply:
Updated.

Reply:
Updated.